pkg/tar/router.go

```diff
@@ -0,0 +1,25 @@
+package tar
+
+import (
+	"archive/tar"
+	"path/filepath"
+)
+
+// Router instructs where to extract a file
+type Router interface {
+	// ExtractPath instructs the path where a file should be extracted
```

Comment:
nit: u can add a `.` in the end of the first sentense or it becomes 
```
ExtractPath instructs the path where a file should be extracted Empty strings instructs to omit the file extraction
```
in doc/IDE highlights